alerter/src/androidTest/java/com/tapadoo/alerter/AlertTest.kt

```diff
@@ -0,0 +1,143 @@
+package com.tapadoo.alerter
+
+import android.annotation.SuppressLint
+import android.graphics.drawable.ColorDrawable
+import android.os.Build
+import android.support.test.filters.LargeTest
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.support.v4.content.ContextCompat
+import android.view.View
+import junit.framework.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Alert Test Case Class
+ *
+ * @author Kevin Murphy, Tapadoo
+ * @since 04/10/2016
+ */
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class AlertTest {
+
+    //Rule which sets the Activity to be used
+    @Rule
+    internal val activityRule = ActivityTestRule<MockActivity>(MockActivity::class.java)
+
+    @Before // Called before each test
+    @Throws(Exception::class)
+    fun setUp() {
+        alert = Alert(activityRule.activity)
+    }
+
+    @Test
+    fun testConstruction() {
+        val alert = Alert(activityRule.activity)
+        Assert.assertNotNull(alert)
+    }
+
+    @Test
+    fun testLayoutElements() {
+        val alert = Alert(activityRule.activity)
+
+        //Ensure all elements are present
+        Assert.assertNotNull(alert.alertBackground)
+        Assert.assertNotNull(alert.title)
+        Assert.assertNotNull(alert.text)
+        Assert.assertNotNull(alert.icon)
+    }
+
+    @Test
+    fun testTitleString() {
+        //Strings
+        alert!!.setTitle(HELLO)
```

Comment:
Careful with `!!` if you know its going to be non-null, set its type to that rather than !!